src/main/java/com/spotify/reaper/resources/view/hierarchy/HCluster.java

```diff
@@ -0,0 +1,93 @@
+package com.spotify.reaper.resources.view.hierarchy;
+
+import com.google.common.base.Function;
+import com.google.common.collect.Collections2;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.spotify.reaper.core.Cluster;
+import com.spotify.reaper.core.RepairRun;
+import com.spotify.reaper.core.RepairSchedule;
+import com.spotify.reaper.core.RepairSegment;
+import com.spotify.reaper.core.RepairUnit;
+import com.spotify.reaper.service.RingRange;
+import com.spotify.reaper.storage.IStorage;
+import com.spotify.reaper.storage.MemoryStorage;
+
+import org.apache.cassandra.repair.RepairParallelism;
+import org.joda.time.DateTime;
+
+import java.math.BigInteger;
+import java.util.Collection;
+import java.util.Set;
+
+import javax.annotation.Nullable;
+
+public class HCluster {
+
+  public static void main(String[] args) throws JsonProcessingException {
```

Comment:
We should keep the testing code outside of main code packages, and only in packages having test in them.